utils/math.go

```diff
@@ -14,3 +14,33 @@ func SumInts(a []int) int {
 	}
 	return n
 }
+
```

Comment:
these all feel pretty silly

Reply:
there's an [`intmath` library](https://pkg.go.dev/github.com/thomaso-mirodin/intmath) with a lot of basic stuff, but the ethos of the language seems to be "if it's easy, do it yourself"